great_expectations_provider/example_dags/example_great_expectations_dag.py

```diff
@@ -3,113 +3,73 @@
 """
 
 import os
-import airflow
+from datetime import datetime
+from pathlib import Path
+
 from airflow import DAG
-from great_expectations_provider.operators.great_expectations import (
-    GreatExpectationsOperator,
-    GreatExpectationsBigQueryOperator,
-)
 
-default_args = {"owner": "Airflow", "start_date": airflow.utils.dates.days_ago(1)}
+from great_expectations_provider.operators.great_expectations import GreatExpectationsOperator
 
-dag = DAG(dag_id="example_great_expectations_dag", default_args=default_args)
+from include.object_configs.example_data_context_config import example_data_context_config
+from include.object_configs.example_checkpoint_config import example_checkpoint_config
 
-# This runs an expectation suite against a sample data asset. You may need to change these paths if you do not have your `data`
-# directory living in a top-level `include` directory. Ensure the checkpoint yml files have the correct path to the data file.
-base_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-data_file = os.path.join(
-    base_path, "include", "data/yellow_tripdata_sample_2019-01.csv"
-)
-ge_root_dir = os.path.join(base_path, "include", "great_expectations")
+"""
+TODO: DataContext configs are here and this is how you specify them and modify them.
```

Comment:
Is this a TODO? It seems like more of a note to me. I'm not clear on what needs to be implemented here.